ui/app/AppLayouts/Chat/views/ChatMessagesView.qml

```diff
@@ -137,7 +146,7 @@ Item {
 
             // HACK: we call `addNewMessagesMarker` later because messages model
             // may not be yet propagated with unread messages when this signal is emitted
-            if (chatLogView.visible && !d.isMostRecentMessageInViewport) {
+            if (chatLogView.visible && Qt.application.state != Qt.ApplicationActive) {
```

Comment:
I believe it should be:
```suggestion
            if (chatLogView.visible && (Qt.application.state != Qt.ApplicationActive || !d.isMostRecentMessageInViewport)) {
```
Otherwise the case 0:48->0:56 https://github.com/status-im/status-desktop/pull/8756 will not work if left side app is constantly in focus.